include/dxc/Support/HLSLOptions.h

```diff
@@ -115,7 +115,7 @@ struct RewriterOpts {
 };
 
 enum class ValidatorSelection : int {
-  Auto,        // Try DXIL.dll; fallback to internal validator
+  Auto,        // Try internal validator; fallback to DXIL.dll
```

Comment:
Does it really fall back?  Or is the fall back logic outside of the code modified here?

Reply:
Good point, I looked outside the code and don't think there is any fallback logic. 
Auto -> Internal